app/services/exportEventsService.js

```diff
@@ -0,0 +1,18 @@
+const logger = require('../utils/logger')('exportEvevntsService');
+const { createExportQueue } = require('../utils/redisQueue');
+
+let eventsQueue;
+
+const initExportEventsQueue = () => createExportQueue('eventsQueue')
```

Comment:
please rename all files and methods from 
*ExportEventsQueue to the *EventsQueue
because export is redunant

Reply:
[Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/18/commits/f63f51034c6dc644adf3b8100c3aa5ca0cc31eae#diff-3dfd1aeed57246722026e05cc36b9a86R6)